features/backend/deleting_account.feature

```diff
@@ -0,0 +1,26 @@
+@ui-user
+Feature: Deleting the account in backend
```

Comment:
Deleting the **user** account ~~in backend~~

This feature should be moved to `/shop/user` folder.